src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin circle($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin circle-dot($size) {
```

Comment:
You have two same mixins, what's the purpose of duplicating it?
Remove one of them

Reply:
i'm fix please check my work))